libs/dh/metering-point/feature-overview/src/lib/dh-customer-overview.component.ts

```diff
@@ -67,46 +69,56 @@ import { DhCustomerContactDetailsComponent } from './dh-customer-contact-details
         <h3>{{ t('title') }}</h3>
       </watt-card-title>
 
-      <div
-        vater-stack
-        direction="row"
-        gap="s"
-        class="protected-address watt-space-inset-squish-s watt-space-stack-m"
-      >
-        <watt-icon size="s" name="warning" />
-        <span class="watt-text-s">{{ t('protectedAddress') }}</span>
-      </div>
-
       <div vater-flex gap="m" direction="row" class="watt-space-stack-m">
         @for (customer of customers(); track customer.id) {
           <div vater-flex gap="s" basis="0" class="customer">
+            <div
+              vater-stack
+              direction="row"
+              gap="s"
+              class="watt-space-inset-squish-s watt-space-stack-m"
+              [class.protected-address]="customer.isProtectedName"
+            >
+              @if (customer.isProtectedName) {
```

Comment:
suggestion: let's move the `if` around the `div` instead. Otherwise it can end up with an empty element

Reply:
I put it there, to avoid the text jumping up if customer name is not protected

Reply:
Can we keep the element more badge-looking (not taking the full width caused by the `flexbox` parent)
`self-align: flex-start` should do it.

![image](https://github.com/user-attachments/assets/193f11e5-d2c1-45b2-bf1d-38625644423f)